app/components/Menu.js

```diff
@@ -0,0 +1,73 @@
+// Note: this is not finished, only shows menu for now
+//menu options not functional
+import React, {
+  Dimensions,
+  StyleSheet,
+  ScrollView,
+  View,
+  Image,
+  Text,
+  Component,
+} from 'react-native';
+const window = Dimensions.get('window');
+const uri = 'http://pickaface.net/includes/themes/clean/img/slide2.png';
+
+const styles = StyleSheet.create({
+  menu: {
+    flex: 1,
+    width: window.width,
+    height: window.height,
+    backgroundColor: 'gray',
+    padding: 20,
+  },
+  avatarContainer: {
+    marginBottom: 20,
+    marginTop: 20,
+  },
+  avatar: {
+    width: 48,
+    height: 48,
+    borderRadius: 24,
+    flex: 1,
+  },
+  name: {
+    position: 'absolute',
+    left: 70,
+    top: 20,
+  },
+  item: {
+    fontSize: 14,
+    fontWeight: '300',
+    paddingTop: 5,
+  },
+});
+
+export default class Menu extends Component {
+  static propTypes = {
```

Comment:
Can we refactor this to Map.propTypes = {} after the class definition? According to @bignell, static properties are ES7 and not yet fully supported.